.github/workflows/licensing.yml

```diff
@@ -0,0 +1,36 @@
+# Copyright (C) 2017-2021  The SymbiFlow Authors.
+#
+# Use of this source code is governed by a ISC-style
+# license that can be found in the LICENSE file or at
+# https://opensource.org/licenses/ISC
+#
+# SPDX-License-Identifier: ISC
+
+name: Licensing
+
+on:
+  push:
+  pull_request:
+
+
+jobs:
+  Checks:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - uses: SymbiFlow/actions/checks@main
```

Comment:
The `checks` should understand things under `third_party` are third_party :-)

Reply:
Can you log a bug upstream about third_party not being ignored?

Reply:
Actually third_party are not ignored. Previously there was a problem with not cloned submodules.